Userland/Applications/PixelPaint/ImageEditor.cpp

```diff
@@ -319,8 +319,8 @@ void ImageEditor::mousemove_event(GUI::MouseEvent& event)
     if (event.buttons() & GUI::MouseButton::Middle) {
         auto delta = event.position() - m_click_position;
         m_pan_origin = m_saved_pan_origin.translated(
-            -delta.x() / m_scale,
-            -delta.y() / m_scale);
+            -delta.x(),
+            -delta.y());
```

Comment:
This seems to be for panning with the middle mouse button and doesn't seem right... Was this intended?

Reply:
Yup. I removed the dividing by scale and then multiplying by scale again, too (see ln 599-600).
There wasn't an easy way to seperate that into its own atomic commit.